src/main/features/claim/routes/check-and-send.ts

```diff
@@ -24,10 +24,10 @@ import { DraftService } from 'services/draftService'
 import { DraftClaim } from 'drafts/models/draftClaim'
 import { Draft } from '@hmcts/draft-store-client'
 
-function getClaimAmountTotal (draft: DraftClaim): Promise<TotalAmount> {
-  return FeesClient.calculateIssueFee(claimAmountWithInterest(draft))
-    .then((feeAmount: number) => {
-      return new TotalAmount(draft.amount.totalAmount(), interestAmount(draft), feeAmount)
+async function getClaimAmountTotal (draft: DraftClaim): Promise<TotalAmount> {
+  return FeesClient.calculateIssueFee(await claimAmountWithInterest(draft))
```

Comment:
Mixing promises and `async->await` doesn't look like a good pattern to me.

I had a bit of a play and this is what I came up with:
It removes one API call as interest was being calculated twice, and it also removes an additional unnecessary totalAmount calculation.
It is still using promises and async->await but they aren't mixed inside the function calls
```
async function getClaimAmountTotal (draft: DraftClaim): Promise<TotalAmount> {
  const interest: number = await interestAmount(draft)
  const totalAmount: number = draft.amount.totalAmount()
  
  return FeesClient.calculateIssueFee(totalAmount + interest)
    .then((feeAmount: number) => new TotalAmount(totalAmount, interest, feeAmount))
}
 ```

REQ